client/state/selectors/test/index.js

```diff
@@ -27,7 +27,9 @@ const RX_JS_EXTENSION = /\.js$/;
 describe( 'selectors', () => {
 	it( 'should match every selector to its default export', () => {
 		each( selectors, ( selector, key ) => {
-			expect( require( '../' + kebabCase( key ) ) ).to.equal( selector );
+			const module = require( '../' + kebabCase( key ) );
+			const defaultExport = module.default ? module.default : module;
+			expect( defaultExport ).to.equal( selector );
```

Comment:
>@aduth I had to update this test to allow global selectors to export "utils" functions for test purpose.

Makes sense 👍  If/when we drop [`babel-plugin-transform-export-extensions`](https://github.com/Automattic/wp-calypso/blob/12559ed9578886d95f7a79f1f4d6d4365d234c4a/.babelrc#L9) ([npm](https://www.npmjs.com/package/babel-plugin-transform-export-extensions)) this can be simplified to:

```js
expect( require( '../' + kebabCase( key ) ).default ).to.equal( selector );
```